.github/workflows/test_clients.yaml

```diff
@@ -29,18 +29,54 @@ jobs:
     env:
       DOTNET_NOLOGO: true
     steps:
-    - name: Check out repo
+    - name: Proto Schemas > Checkout Repository
       uses: actions/checkout@v3
 
-    - name: Check out google-cloudevents-dotnet
+    - name: .NET Library > Checkout Repository
       uses: actions/checkout@v3
       with:
         repository: googleapis/google-cloudevents-dotnet
         path: google-cloudevents-dotnet
+
     - name: Setup .NET 6
       uses: actions/setup-dotnet@v3
       with:
         dotnet-version: 6.0.x
 
     - name: Run smoke tests
       run: google-cloudevents-dotnet/validate-schema.sh .
+
+  java:
+    runs-on: ubuntu-latest
+    env:
+      LIBRARY_CHECKOUT_PATH: library
+      DATA_SOURCE_CHECKOUT_PATH: google-cloudevents
+    steps:
+      - name: Java Library > Checkout Repository
+        uses: actions/checkout@v3
+        with:
+          repository: googleapis/google-cloudevents-java
+          path: ${{ env.LIBRARY_CHECKOUT_PATH }}
+          ref: snowpea-build
+
+      - name: Proto Schemas > Checkout Repository
```

Comment:
I wonder if we could do a single checkout of proto schemas and reuse across all languages?

Reply:
I don't think it's worth the work trying to create a parent job to checkout and have child jobs uses that one source. Are you concerned or just trying to minimize each language setup?

Reply:
Just thinking about avoiding redundant actions and per-language complexity. But agreed, refactoring for that is a good future improvement.